tests/util/gateway-api.sh

```diff
@@ -14,16 +14,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+source "content/en/boilerplates/snips/gateway-api-support.sh"
 
-K8S_GATEWAY_API_CRDS="github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.5.0"
 GATEWAY_API="true"
 
 function install_gateway_api_crds() {
-    kubectl kustomize "${K8S_GATEWAY_API_CRDS}" | kubectl apply -f -
+    bpsnip_gateway_api_support_install_crds
 }
 
 function remove_gateway_api_crds() {
-    kubectl kustomize "${K8S_GATEWAY_API_CRDS}" | kubectl delete -f -
+    bpsnip_gateway_api_support_install_crds
```

Comment:
This boilerplate snip is installing the crds instead of deleting them. It is also not quite right for the above `install_gateway_api_crds` either, since it is checking if they exist before installing, instead of just installing. Maybe you should add another dummy boilerplate just for use by this function, e.g., something like this (`boilerplates/gateway-api-util.md`:
```
---
---

{{< text syntax=plain snip_id=crd-version >}}
{{< k8s_gateway_api_version >}}
{{< /text >}}

```

Then you can change gateway-api.sh to this:

```
K8S_GATEWAY_API_CRDS="github.com/kubernetes-sigs/gateway-api/config/crd?ref=${bpsnip_gateway_api_util_crd_version}"
```

Reply:
Alternatively, if it doesn't slow things down too much, can `gateway-api.sh` just read the version value from `data/args.yaml`?

Reply:
We could do that, but I think the former is a better precedence if we have to do something like this again.